docs/reference/cli/connect.md

```diff
@@ -36,9 +36,12 @@ tailpipe connect --from 2025-01-01
 ```
 
 > [!NOTE]
-> You could use this connection string with DuckDB:
+> You can use this connection string with DuckDB to directly query the Tailpipe database.
+To ensure compatibility with tables that include JSON columns, make sure you’re using DuckDB version 1.1.3 or later.
```

Comment:
Is that because the JSON extension is installed/enabled by default?  Could you also install it in an older version, or disable it in a newer version?  Are there any other extensions that are required/recommended?